Assets/Scripts/Notebook/Notebook Interactions/Counter.cs

```diff
@@ -22,11 +21,9 @@ private void Awake()
         {
             ResetCount();
         }
+
         private void OnEnable()
         {
-            _chapter = gameObject.GetComponent<Chapter>();
-            if (_chapter == null) UnityEngine.Debug.LogError("Trying to access the chapter of this gameobject, but there is none.");
```

Comment:
no longer using a chapter? Why not?

Reply:
dragging in a reference to the TMP directly and counter.cs is on the home chapter so this script is only enabled when home is turned on